MAINTAINERS.md

```diff
@@ -1,14 +1,16 @@
-# OpenSearch SQL Maintainers
+## Overview
 
-## Maintainers
+This document contains a list of maintainers in this repo. See [opensearch-project/.github/RESPONSIBILITIES.md](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#maintainer-responsibilities) that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
 
-| Maintainer | GitHub ID | Affiliation |
-| --------------- | --------- | ----------- |
-| Anirudha (Ani) Jadhav | [anirudha](https://github.com/anirudha) | Amazon |
-| Peng Huo | [penghuo](https://github.com/penghuo) | Amazon |
-| Chen Dai | [dai-chen](https://github.com/dai-chen) | Amazon |
-| Chloe Zhang | [chloe-zh](https://github.com/chloe-zh) | Amazon |
-| Nick Knize | [nknize](https://github.com/nknize) | Amazon |
-| Charlotte Henkle | [CEHENKLE](https://github.com/CEHENKLE) | Amazon |
-| Max Ksyunz |  [MaxKsyunz](https://github.com/MaxKsyunz) | BitQuill |
-| Yury Fridlyand | [Yury-Fridlyand](https://github.com/Yury-Fridlyand) | BitQuill |
\ No newline at end of file
+## Current Maintainers
+
+| Maintainer            | GitHub ID                                           | Affiliation |
+| --------------------- | --------------------------------------------------- | ----------- |
+| Anirudha (Ani) Jadhav | [anirudha](https://github.com/anirudha)             | Amazon      |
+| Peng Huo              | [penghuo](https://github.com/penghuo)               | Amazon      |
+| Chen Dai              | [dai-chen](https://github.com/dai-chen)             | Amazon      |
+| Chloe Zhang           | [chloe-zh](https://github.com/chloe-zh)             | Amazon      |
+| Nick Knize            | [nknize](https://github.com/nknize)                 | Amazon      |
+| Charlotte Henkle      | [CEHENKLE](https://github.com/CEHENKLE)             | Amazon      |
+| Max Ksyunz            | [MaxKsyunz](https://github.com/MaxKsyunz)           | BitQuill    |
+| Yury Fridlyand        | [Yury-Fridlyand](https://github.com/Yury-Fridlyand) | BitQuill    |
```

Comment:
@dblock, could you please update affiliation for me and @MaxKsyunz? Bit Quill was acquired by Improving recently: [link](https://improving.com/newsroom/improving-acquires-bitquill).
```suggestion
| Max Ksyunz            | [MaxKsyunz](https://github.com/MaxKsyunz)           | Improving   |
| Yury Fridlyand        | [Yury-Fridlyand](https://github.com/Yury-Fridlyand) | Improving   |
```

Reply:
Can we keep this change at format only just for consistency? You guys can make the affiliation changes and others later? LMK if that's ok.